html/index.py

```diff
@@ -738,7 +742,7 @@
                 
                 btn_send_tx.disable(true);
                 show_progress("Sending coins... This can take a while for big amount...");
-                app_hub.send_tx(amount, address, payment_id, priority, mixin, tx_desc, $('#checkbox_save_address').is(":checked"));
+                app_hub.send_tx(amount, address, payment_id, priority, mixin, tx_desc, $('#checkbox_save_address').is(":checked"), sweep_all););

```

Comment:
Should be:

`app_hub.send_tx(amount, address, payment_id, priority, mixin, tx_desc, $('#checkbox_save_address').is(":checked"), sweep_all);`